modules/gnome/nixos.nix

```diff
@@ -2,41 +2,59 @@
   lib,
   pkgs,
   config,
+  mkTarget,
   ...
 }:
+mkTarget {
+  name = "gnome";
+  humanName = "GNOME and GDM";
+  autoEnable =
+    config.services.xserver.desktopManager.gnome.enable
+    || config.services.xserver.displayManager.gdm.enable;
```

Comment:
Didn't these options get recently moved out of the `xserver` namespace?

I believe this will generate mkRenamedOptionModule warnings.

Reply:
Indeed, the existing impl is using:

```
          config.services.desktopManager.gnome.enable 
           || config.services.displayManager.gdm.enable
```